google/cloud/dataproc/README.md

```diff
@@ -38,24 +42,34 @@ this library.
 
 <!-- inject-quickstart-start -->
 ```cc
-#include "google/cloud/dataproc/ EDIT HERE .h"
-#include "google/cloud/project.h"
+#include "google/cloud/dataproc/cluster_controller_client.h"
+#include "google/cloud/common_options.h"
 #include <iostream>
 #include <stdexcept>
 
 int main(int argc, char* argv[]) try {
-  if (argc != 2) {
-    std::cerr << "Usage: " << argv[0] << " project-id\n";
+  if (argc != 3) {
+    std::cerr << "Usage: " << argv[0] << " project-id region\n";
     return 1;
   }
 
+  namespace gc = ::google::cloud;
   namespace dataproc = ::google::cloud::dataproc;
-  auto client = dataproc::Client(dataproc::MakeConnection(/* EDIT HERE */));
 
-  auto const project = google::cloud::Project(argv[1]);
-  for (auto r : client.List /*EDIT HERE*/ (project.FullName())) {
-    if (!r) throw std::runtime_error(r.status().message());
-    std::cout << r->DebugString() << "\n";
+  std::string const project_id = argv[1];
+  std::string const region = argv[2];
+
+  gc::Options options;
+  if (region != "global") {
```

Comment:
Can we hard-code the region to "global" and just ask for the project-id in the command-line?

Reply:
I would not. Setting `region="global"` does not list all clusters in any region. "global" itself is a region (like "us-central1"), so it would only list clusters that were created in the "global" region. When you create a cluster using the UI, the default region is something like "us-central1", not "global".

Reply:
Yeah... I was trying to simplify the code / save vertical space.

In any case, this suggests we need a helper function (for a future PR), something like:

```cc
namespace google::cloud::dataproc {
gc::Options WithRegionalEndpoint(Options options, std::string const& region);
}
```

Reply:
Ack, I see the issue you opened.